### DISCUSSION TITLE
Can't, for the life of me, figure out argument to --details!

### DISCUSSION THREAD

**Discussion by djhunter67** [pls-rs/pls#26] [Help Q&A] [answered]
How do i use the --details argument with anything other than blank or with all?
![pls_discussion](https://user-images.githubusercontent.com/25860608/166167607-4c3a3c5c-9222-498f-af02-8e3b7d98dc98.gif)

**Comment by dhruvkb** [comment, marked as answer]
To specify multiple detail columns, you can repeat the `-d` flag.

```
$ pls -u decimal -d type -d size
```

Hopefully, the [documentation for this feature](https://dhruvkb.github.io/pls/features/details.html) can help as it explains this in more detail.
